_posts/en/newsletters/2023-09-06-newsletter.md

```diff
@@ -0,0 +1,129 @@
+  elements.  However, typical Bitcoin transactions consist largely of
+  uniform elements---data that looks random---like public keys and hash
+  digests.
+
+    Briar's proposal addresses this using several approaches:
+
+    - For the parts of a transaction where an integer is currently
+      represented by 4 bytes (e.g., transaction version and outpoint
+      index), these are replaced by a variable-length integer that can
+      be as small as 2 bits.
+
+    - For the uniformly distributed 32-byte outpoint txid in each input,
+      these are replaced by a referenced to the location of that
+      transaction in the block chain using its block height and location
+      within the block, e.g.  `123456` and `789` would indicate the
+      789th transaction in block 123,456.  Because the block at a
+      particular height can change due to a block chain reorganization,
+      this method is only used when the referenced transaction has at
+      least 100 confirmations.
+
+    - For P2WPKH transactions where the witness structure needs to
+      include a signature plus a public key that is typically 33 bytes,
+      the public key is omitted and a technique of reconstructing it
+      from signature is used.
+
+    Some other techniques are used to save a few extra bytes in typical
+    transactions.  The general downside of the proposal is that
+    converting a compressed transaction back into something that full
+    nodes and other software can use requires more CPU, memory, and I/O
+    than processing a regular serialized transaction.  That means
+    high-bandwidth connections will likely continue to use the regular
+    transaction format and only low-bandwidth transmission will use
+    compressed transactions.
+
+    The idea received a moderate amount of discussion, mostly around
+    ideas for saving a small amount of additional space per input.
+
+- **Privacy enhanced co-signing:** Nick Fallow [posts][fallow cosign] to
+  the Bitcoin-Dev mailing list about how a [scriptless threshold
+  signature scheme][topic threshold signature] like [FROST][] could
+  improve the privacy of people who use co-signing services.  A typical
+  user of a co-signing service has multiple signing keys that are stored
+  separately for security; but, to simplify normal spending, they also
+  allow their outputs to spent by a combination of some of their keys
+  plus one or more keys held by one or more service providers who only
+  sign after authenticating the user in some way.  The user can bypass
+  the service provider if they need to, but the service provider makes
+  operations easier in most cases.
+
+    With scripted threshold signature schemes like 2-of-3
+    `OP_CHECKMULTISIG`, the service's public key must be associated with
+    the output being spent, so any service will be able to find the
+    transactions it signed by looking at onchain data, allowing it
+    accumulate data about its users.  Worse, all currently used protocols
```

Comment:
```suggestion
    accumulate data about its users.  Worse, all current protocols
```
(or "currently-used")

Reply:
Are you sure "currently-used protocols" is correct?  I thought we didn't hyphenate adverbs ending in -ly.

I don't think "current protocols" by itself is sufficient, as blind signing is a current protocol that just isn't used.